README.md

```diff
@@ -15,7 +15,7 @@ To learn more about Xenko, visit [xenko.com](https://xenko.com/).
 
 Xenko is covered by [MIT](LICENSE.md), unless stated otherwise (i.e. for some files that are copied from other projects).
 
-You can find the list of third party projects [THIRD PARTY.md](here).
+You can find the list of third party projects [THIRD PARTY.md](THIRD%20PARTY.md).
```

Comment:
I think it's better to fix`[THIRD PARTY.md](here)` to `[here](THIRD PARTY.md)`, than `[THIRD PARTY.md](THIRD PARTY.md)`

Reply:
That's more natural. Thank you.